apps/video/view.html

```diff
@@ -27,26 +29,31 @@
             <h1 id="video-title"></h1>
             <button id="save" data-l10n-id="save" hidden></button>
           </gaia-header>
-
-          <footer id="videoBar">
-            <div id="timeSlider">
-              <span id="elapsed-text"></span>
-              <div id="slider-wrapper">
-                <div id="elapsedTime" class="progress"></div>
-                <div id="bufferedTime" class="progress"></div>
-                <div id="timeBackground" class="progress"></div>
-                <button id="playHead"></button>
+          <div id="hide-controls" role="button"
+               data-l10n-id="hide-controls-button"></div>
+          <div role="toolbar">
+            <footer id="videoBar" role="presentation">
```

Comment:
I think it would be helpful to include a comment regarding the "presentation" role and why it's being used here.